src/mobile/package.json

```diff
@@ -25,10 +29,11 @@
     "react-native-elements": "3.4.2",
     "react-native-safe-area-context": "3.3.2",
     "react-native-screens": "3.10.1",
+    "react-native-url-polyfill": "1.3.0",
     "react-native-web": "0.17.1"
   },
   "devDependencies": {
-    "@babel/core": "7.17.5",
+    "@babel/core": "7.12.9",
```

Comment:
Is this needed for it to work? If not, let's leave it to Renovate to update it in a separate pr.

Reply:
We did not have `renovate` coming into `src\mobile` unfortunately

Reply:
https://github.com/Seneca-CDOT/telescope/commit/b4d81f831cede711e55ee2642402f95da8af4055

Reply:
Got it !